src/NServiceBus.Core/Routing/RoutingPluginSettingsExtensions.cs

```diff
@@ -19,6 +20,16 @@ public static UnicastRoutingTable RoutingTable(this FeatureConfigurationContext
         }
 
         /// <summary>
+        /// Returns the publishers table.
```

Comment:
should we add a note that this API is only relevant when using message driven pubsub transports?